Code-4-Community/speak-for-the-trees-frontend
# Neighborhood Names Fix: Styling and Updates

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/925293934/views/18904377/pulses/1094380448)



Neighborhood names and outlines have been added to assist users with identifying individual neighborhoods by their long or shorthand names.

## This PR



Property names called using feature.getProperty have to be lower caps (neighborhood_id as opposed to Neighboorhood_ID) and features may need to be iterated through individually (using a neighborhoodlayer.forEach()) if trying to implement neighborhood-level changes.

## Screenshots

 

## Verification Steps

With serviceMain running, npm start and view the map upon loading. Neighboorhood names and their respective outlines should be present in green for outlines and white for names.
